### PR TITLE
Refactor dirmult_ttest function signature and fixed documentation...

### PR DESCRIPTION
**Pull request by code-wolf-byte**
…to match RST formatting.

Please complete the following checklist:

[ * ] I have read the [contribution guidelines](https://scikit.bio/contribute.html).

[ * ] I have documented all public-facing changes in the [changelog](https://github.com/scikit-bio/scikit-bio/blob/main/CHANGELOG.md).

[ * ] This pull request includes code, documentation, or other content derived from external source(s). If this is the case, ensure the external source's [license](https://github.com/scikit-bio/scikit-bio/blob/main/LICENSE.txt) is compatible with scikit-bio's license. Include the license in the licenses directory and add a comment in the code giving proper attribution. Ensure any other requirements set forth by the license and/or author are satisfied.

It is your responsibility to disclose code, documentation, or other content derived from external source(s). If you have questions about whether something can be included in the project or how to give proper attribution, include those questions in your pull request and a reviewer will assist you.
[ * ] This pull request does not include code, documentation, or other content derived from external source(s).

Note: [This document](https://scikit.bio/devdoc/review.html) may also be helpful to see some of the things code reviewers will be verifying when reviewing your pull request.

### CODE REVIEW

File: skbio/stats/composition.py [hunk 1]
```diff
@@ -1757,7 +1757,7 @@ def dirmult_ttest(
     To perform the t-test, we first fit a Dirichlet-multinomial distribution
```
Comment (mataton) [inline skbio/stats/composition.py:1757]: The "t-test" in this line should also be italicized.

File: skbio/stats/composition.py [hunk 2]
```diff
@@ -1793,41 +1793,41 @@ def dirmult_ttest(
...
         from each of the posterior draws.
 
-        `"CI(97.5)"` is the 97.5% quantile of the log2-fold change. The
-        reported `CI(97.5)` is the 97.5% quantile of all of the log2-fold
+        ``CI(97.5)`` is the 97.5% quantile of the log2-fold change. The
+        reported ``CI(97.5)`` is the 97.5% quantile of all of the log2-fold
         changes computed from each of the posterior draws.
 
-        `"pvalue`" is the pvalue of the t-test. The reported `pvalue` is the
+        ``p-value`` is the pvalue of the *t*-test. The reported ``*p*-value`` is the
```
Comment (mataton) [inline skbio/stats/composition.py:1817]: `pvalue` here refers to the variable name which the function returns, so it should be left as ``pvalue`` with two tick marks (`) on either side, with no hyphen or italics. Same with `qvalue` below.

**Comment by mataton** [comment]
@code-wolf-byte Good job! I left a couple of comments. It looks like there is a linting issue, this can be fixed by running `ruff check .` in the command line before you commit, or by using the pre-commit hooks mentioned [here](https://scikit.bio/contribute.html#style-code). Additionally, it looks like you are submitting from your `main` branch. Typically it is considered best practice to write your code on a separate branch, and submit a pull request which references that separate branch. That way, your local main branch stays up to date and not in conflict with the main branch of the project. [Here](https://www.atlassian.com/git/tutorials/using-branches) is some more information about git branches.

**Comment by qiyunzhu** [comment]
@code-wolf-byte Thanks for your contribution! It looks like the linting thing is minor and can be fixed quickly. Looking forward to your update!

**Pull request closed without merging**